### PR TITLE
fix: Fixed a wrong link to supported python versions in `contributing.md` file

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# What does this PR do?
Fixes a wrong link to supported Python versions in [`contributing.md`](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md) file.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Core library:
- General functionalities: @sayakpaul

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -355,7 +355,7 @@ You will need basic `git` proficiency to be able to contribute to
 manual. Type `git --help` in a shell and enjoy. If you prefer books, [Pro
 Git](https://git-scm.com/book/en/v2) is a very good reference.
 
-Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L265)):
+Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L270)):
```
Comment (sayakpaul) [inline CONTRIBUTING.md:358]: Actually, this should ideally be a permalink.

Comment (Sai-Suraj-27) [reply CONTRIBUTING.md]: Yeah, it makes more sense. I updated the link to a permalink. Please check now. Thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -355,7 +355,7 @@ You will need basic `git` proficiency to be able to contribute to
 manual. Type `git --help` in a shell and enjoy. If you prefer books, [Pro
 Git](https://git-scm.com/book/en/v2) is a very good reference.
 
-Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L265)):
+Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/42f25d601a910dceadaee6c44345896b4cfa9928/setup.py#L270)):
```
Comment (yiyixuxu) [inline CONTRIBUTING.md:358]:
```suggestion
Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L270)):
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (standardAI) [reply CONTRIBUTING.md]: It will be only needed to update this permalink when supported Python versions are changed. Make sense :+1:.

**Review by standardAI — APPROVED** [review]
Thanks for fixing!
Actually, it was me to make it a temporary link last time 😅.

**Review by sayakpaul — APPROVED** [review]

**Pull request merged**